source/reference/configuration.txt

```diff
@@ -100,6 +100,11 @@ Options
    To set the ``loadLevel`` setting from the command line, see the
    :option:`--loadLevel` option.
 
+   .. warning::
+
+      Setting the ``loadLevel`` higher than the default may negatively impact 
+      the performance of the destination cluster. 
+
```

Comment:
(minimalism)
Consider the following minor tweaks to make the sentence a bit simpler. Something like:
"Setting ``loadLevel`` higher than the default may negatively impact the destination cluster performance." 

(question/level of detail) 
Does it make sense contextually to say what the default is?

Reply:
Thanks for the feedback @sarah-olson-mongodb ! I changed the warning to an includes and took your suggestions! I think it does make sense to specify what the default load level is.